ingeniamotion/communication.py

```diff
@@ -820,7 +821,7 @@ def scan_servos_ethercat(
         self,
         interface_name: str,
     ) -> List[int]:
-        """Scan a network adapter to get all connected EtherCAT slaves.
+        """Scan a network adapter to get all connected EtherCAT slaves - CoE.
```

Comment:
scan_servos_ethercat does not use SDOs or PDOs, so it is not only CoE. For example, you can find a drive in boot to update the FW with FoE. This applies to all the scan functions.